styles/reference.module.scss

```diff
@@ -4,8 +4,17 @@
   padding: var(--referencePagePadding);
```

Comment:
Large screen vs mobile (375 x 667). On large screen, Braille and semaphore symbols cover almost the whole height of each row. However on mobile, only a part of the height of each row is covered. Not sure if this difference is intentional.

<img width="409" height="643" alt="image" src="https://github.com/user-attachments/assets/7ed578a6-a73d-4651-ab57-d6a2b1590f3c" />
<img width="374" height="658" alt="image" src="https://github.com/user-attachments/assets/ae46c5b5-237e-47ee-8b39-7cc674c64c68" />